Kitodo/src/main/java/de/sub/goobi/forms/AktuelleSchritteForm.java

```diff
@@ -170,7 +170,8 @@ public String filterAlleStart() {
     }
 
     /**
-     * This method initializes the task list without any filter whenever the bean is created.
+     * This method initializes the task list without any filter whenever the
```

Comment:
I use our code formatted, and it formats whole file on which it is applied, not only touched lines.